src/tasks/task-contact-force-equality.cpp

```diff
@@ -17,12 +17,122 @@
 
 #include <Eigen/Dense>
 #include <pinocchio/multibody/model.hpp>
+#include "tsid/tasks/task-contact-force-equality.hpp"
 
-namespace tsid
+namespace tsid {
+namespace tasks {
+
+using namespace tsid::math;
+using namespace std;
+
+TaskContactForceEquality::TaskContactForceEquality(const std::string & name, RobotWrapper & robot,
+                                                   const double dt, const std::string & contactName):
```

Comment:
Instead of taking the contact name you could directly take a reference to the Contact object in the constructor, which you can then you to retrieve the ForceGeneratorMatrix